pre_commit_hooks/check_added_large_files.py

```diff
@@ -21,11 +21,19 @@ def lfs_files() -> Set[str]:
     return set(json.loads(lfs_ret)['files'])
 
 
-def find_large_added_files(filenames: Sequence[str], maxkb: int) -> int:
+def find_large_added_files(
+    filenames: Sequence[str], maxkb: int,
+    enforce_all: Optional[bool] = False,
+) -> int:
     # Find all added files that are also in the list of files pre-commit tells
     # us about
     retv = 0
-    for filename in (added_files() & set(filenames)) - lfs_files():
+    filename_set = set(filenames)
+    if not enforce_all:
+        filename_set &= added_files()
+        filename_set -= lfs_files()
```

Comment:
this should still exclude lfs_files even with `--enforce-all`, they aren't relevant to the repository size